src/core/env/ClientContext.tsx

```diff
@@ -0,0 +1,62 @@
+'use client';
```

Comment:
just looked this up briefly, since it's the first time I see this being used. Is there potential for using this elsewhere also? (not specifically related to this pR, just curious)

Reply:
Yeah this is a feature of NEXT.js and more specifically "React Server Components". When we start using the app router, we will see much more of this, as a way to indicate components that should not be rendered using RSC.